product_tax_multicompany_default/tests/test_product_tax_multicompany.py

```diff
@@ -12,8 +12,8 @@ def setUpClass(cls):
         cls.company_1 = cls.env["res.company"].create({"name": "Test company 1"})
         cls.company_2 = cls.env["res.company"].create({"name": "Test company 2"})
         group_account_manager = cls.env.ref("account.group_account_manager")
-        ResUsers = cls.env["res.users"]
```

Comment:
Is this renaming due to anything? If not, please keep it for reducing the noisy diff

Reply:
I changed all the names of the variables that will store model objects just to comply with OCA conventions. In one part it says: "Don't use CamelCase for model variables".
Should I leave it as it was anyway?

Reply:
Yes, please, that guideline should be adjusted, as this convention for model objects is common in Odoo upstream, as it represents somehow a "class".